cmd/prometheus/main.go

```diff
@@ -235,9 +237,9 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 			case "delayed-compaction":
 				c.tsdb.EnableDelayedCompaction = true
 				level.Info(logger).Log("msg", "Experimental delayed compaction is enabled.")
-			case "utf8-names":
-				model.NameValidationScheme = model.UTF8Validation
-				level.Info(logger).Log("msg", "Experimental UTF-8 support enabled")
+			case "legacy-names":
```

Comment:
Does this mean 2.x has `--enable-feature=utf8-names` and 3.x will have `--enable-feature=legacy-names` with utf8 on by default?

Reply:
Using legacy-names doesn't sound like an experimental-feature 😅 

Would it make sense to move this configuration to another flag? Or maybe configuration file?

Reply:
2.x will have a feature called utf8-names available but it will not be set by default.  3.x will not have a feature flag for it, utf8 will be default-on and it can be disabled through the global or scrapeconfig flags